sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -21,7 +21,7 @@ grammar SqlBase;
    * When false, INTERSECT is given the greater precedence over the other set
    * operations (UNION, EXCEPT and MINUS) as per the SQL standard.
    */
-  public boolean legacy_setops_precedence_enbled = false;
+  public boolean legacy_setops_precedence_enabled = false;
```

Comment:
Oh hm I wonder if this has any implications elsewhere, like does this affect any non-private generated code?

Reply:
GitHub says maybe: https://github.com/search?q=legacy_setops_precedence_enbled&type=code

I'm going to drop this for now, we can revisit it separately.